configs/local.yaml

```diff
@@ -8,6 +8,12 @@ app:
   reload: True
   thread_limiter_total_tokens: 40
 
+http:
```

Comment:
isn't `app` above have the same config values ?

Reply:
`app` runs on 8000 and `http` on 8080